src/AutoRest.CSharp/LowLevel/Output/AspDotNetExtension.cs

```diff
@@ -0,0 +1,160 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using AutoRest.CSharp.Common.Output.Builders;
+using AutoRest.CSharp.Generation.Types;
+using AutoRest.CSharp.Generation.Writers;
+using AutoRest.CSharp.Input;
+using AutoRest.CSharp.Input.Source;
+using AutoRest.CSharp.Output.Models;
+using AutoRest.CSharp.Output.Models.Shared;
+using AutoRest.CSharp.Output.Models.Types;
+using AutoRest.CSharp.Utilities;
+using Azure.Core;
+using Azure.Core.Extensions;
+
+namespace AutoRest.CSharp.LowLevel.Output
+{
+    internal class AspDotNetExtension : TypeProvider
```

Comment:
It is not AspDotNet specific, I would call it `ClientExtesion` or `ServiceCollectionExtenstion`

Reply:
It is aspdotnet specific